CFMS/urls.py

```diff
@@ -20,6 +20,7 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('workorders/', include('workorders.urls')),
```

Comment:
The same nesting of the parts urls should be done with the work order urls.
Following this, you can just use a list of all the parts and work orders for the nav bar

`path('workorders/all', workorders_all, name='workorders_all')`
`path('parts/all', parts_all, name='parts_all')`